independent-publisher-connectors/GitHub Gists/apiProperties.json

```diff
@@ -0,0 +1,27 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "github",

```

Comment:
@troystaylor Thank you so much for submitting a new connector. May I ask you please to change the identityProvider. We are getting the errors below.
##[error]@{level=Error; message=Value "github" is not defined in enum.; valuePath=line: 0 position: 0 path: 'properties.connectionParameters.token.oAuthSettings.identityProvider'; value=github}
##[error]@{level=Error; message=JSON is valid against no schemas from 'oneOf'.; valuePath=line: 0 position: 0 path: 'properties.connectionParameters.token'; value=}
##[error]@{level=Critical; message=Invalid value for identity provider.; valuePath=$.properties.connectionParameters.token.oAuthSettings.identityProvider; value=github}

can you please fix. Thank you so much.

Reply:
I have fixed this. Can you tell me if I have missed in the documentation that OOTB OAuth identity providers are not acceptable for certified connectors? And/or if their acceptable is on the roadmap?
CC: @jopanchal